util/src/main/java/io/grpc/util/AdvancedTlsX509TrustManager.java

```diff
@@ -43,30 +44,38 @@
 /**
  * AdvancedTlsX509TrustManager is an {@code X509ExtendedTrustManager} that allows users to configure
  * advanced TLS features, such as root certificate reloading, peer cert custom verification, etc.
+ * We expect only one of {@link AdvancedTlsX509TrustManager#useSystemDefaultTrustCerts()},
```

Comment:
Optional presentation suggestion: Consider structuring the Javadoc as:
```
AdvancedTlsX509TrustManager is an {@code X509ExtendedTrustManager} that allows users to configure
advanced TLS features, such as root certificate reloading, peer cert custom verification, etc.

For Android users: this class is only supported in API level 24 and above.

We expect only one of {@link AdvancedTlsX509TrustManager#useSystemDefaultTrustCerts()},
{@link AdvancedTlsX509TrustManager#updateTrustCredentials(X509Certificate[])},
 {@link AdvancedTlsX509TrustManager#updateTrustCredentialsFromFile(File, long, TimeUnit,
 ScheduledExecutorService)},
{@link AdvancedTlsX509TrustManager#updateTrustCredentialsFromFile(File)} methods to be called
 after instantiation of this class.


```

Reply:
I like the idea of extra empty LOCs for separation (pushed). But I prefer the current order because of

1. Overall description
2. Impl notes (for all users, including Android)
3. Extra note (for Android users only)

Reply:
SG, just an optional suggestion. :)